lib/DDG/Spice/MovieRuntime.pm

```diff
@@ -0,0 +1,17 @@
+package DDG::Spice::MovieRuntime;
+# ABSTRACT: Search for movie runtime
+
+use strict;
+use DDG::Spice;
+
+triggers startend => "movie runtime", "movie runnning time", "movie length";
+
+spice to => 'http://www.omdbapi.com/?t=$1&plot=short&r=json';
```

Comment:
Yes, and we already have our own API Key. You should be able to register and get one as well